tests/src/Functional/InstallationTest.php

```diff
@@ -69,4 +69,15 @@ public function testKnownResources() {
     );
   }
 
+  public function testGraphQLQuery() {
+    $query = '{"query":"query{nodeQuery{entities{entityLabel ... on NodeRecipe { fieldIngredients }}}}","variables":null}';
```

Comment:
The GraphQL module comes with test base classes that would simplify this test case greatly.

Reply:
https://github.com/drupal-graphql/graphql/blob/8.x-3.x/tests/src/Traits/GraphQLFileTestTrait.php
https://github.com/drupal-graphql/graphql/blob/8.x-3.x/tests/src/Kernel/GraphQLFileTestBase.php

Reply:
Used here for instance: https://github.com/drupal-graphql/graphql/blob/8.x-3.x/modules/graphql_core/tests/src/Kernel/Entity/EntityByIdTest.php

Reply:
@fubhy thanks for the review. I'm wondering if there is any updated doc about mutations. Can't make it work on my local project.

Reply:
From my point of view having some level of smoke testing, just to know whether like our core.extension is right is helpful. We don't have to overdo it, of course

Reply:
I'm personally really :+1: having one test in there!